### PR TITLE
Engine update

### PR DESCRIPTION
**Pull request by nkstlrv**
**Updates:**

- Server initial setup;
- SMS endpoint for Webhook receiving;
- Schema for payload added;
- FUB class for it's utils added;

### CODE REVIEW

File: src/utils/fub_utils.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        if status_code == 200: 
+            result["success"] = True
+
+        return result
+
+
+
+if __name__ == "__main__":
+    ...
```
Comment (sakhaline) [inline src/utils/fub_utils.py:74]: Add empty line at the end of the file

File: src/views/sms_views.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            buyer_phone = buyer_phones[0]["value"] if buyer_phones and type(buyer_phones) == list else None
+
+            if buyer_phone:
+
+                # send sms
+                ...
+                    
+
+            
+
```
Comment (sakhaline) [inline src/views/sms_views.py:35]: Why is so many empty lines?

File: src/utils/fub_utils.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            }
+        )
+
+        status_code = response.status_code
+        logger.info(f"{self.get_buyer.__name__} -- FUB API STATUS CODE - {status_code}")
+
+        try:
+            response["data"] = response.json()
+            logger.info(f"{self.get_buyer.__name__} -- FUB API RESPONSE - {response['data']}")
+        except Exception:
```
Comment (sakhaline) [inline src/utils/fub_utils.py:63]: It is bad practice to use global Exception, try to define what kind of Exception could be raised here and handle it!

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,5 @@ venv
 .env
 env
 *pycache*
+*.log
+sandbox.py
```
Comment (sakhaline) [inline .gitignore:7]: The same...
(empty line)

**Review by sakhaline — APPROVED** [review]

File: src/utils/fub_utils.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+            logger.exception(f"{self.get_buyer.__name__} -- !!! FUB API ERROR")
+
+        if status_code == 200: 
+            result["success"] = True
+
+        return result
+
```
Comment (sakhaline) [inline src/utils/fub_utils.py:70]: Why is so many lines here???....

**Review by sakhaline — APPROVED** [review]

File: src/views/sms_views.py (hunk 2 above)
Comment (nkstlrv) [reply src/views/sms_views.py:35]: Good code requires extra space to breath!

**Pull request closed without merging**